tests/functional/spec/navigation.spec.js

```diff
@@ -151,6 +151,15 @@ describe('Navigation', function() {
       });
   });
 
+  it('Given I have a section that can not be reached from the start of the survey, When I click on the navigation link,' +
+    ' Then the first block of that section should be displayed', function() {
+    return helpers.openQuestionnaire('test_navigation_routing.json').then(() => {
```

Comment:
Would it not be better to just do a beforeEach to open the questionnaire rather then having to do it for every scenario for the Navigation?

Reply:
@MIslam92 this questionnaire is only used once.

Reply:
@jonnyshaw89 Sorry, I was referring to when the test_navigation.json was used my bad